Dockerfile

```diff
@@ -15,6 +18,9 @@ RUN pub get
 ADD app /project/app
 RUN pub get --offline
 
+# Flutter setup runs only if GAE_SERVICE=="analyzer"
+RUN /project/app/script/setup-flutter.sh
```

Comment:
Good catch! I wouldn't put that extra 400MB in the image, rather I did a facelift, PTAL: I've remove `setup-flutter.sh` from `Dockerfile`, and calling it from `analyzer.dart` instead.

Env variables should be initialized at that point, and there is minimal overhead compared to the setup in Docker.